clients/fides-js/src/lib/preferences.ts

```diff
@@ -63,12 +69,7 @@ export const updateConsentPreferences = ({
     user_geography: userLocationString,
     method: consentMethod,
   };
-  patchUserPreferenceToFidesServer(
-    privacyPreferenceCreate,
-    fidesApiUrl,
-    cookie.identity.fides_user_device_id,
-    debug
-  );
+  patchUserPreferenceToFidesServer(privacyPreferenceCreate, fidesApiUrl, debug);
```

Comment:
note: the third argument for `fides_user_device_id` was not being used in the `patch` function. it's actually included in `privacyPreferenceCreate`, so I removed it